main.go

```diff
@@ -328,7 +333,7 @@ func createMeasurements(
 	logger *log.Logger,
 	orcWorkflow cfWorkflow.CfWorkflow,
 	pushWorkFlowGeneratorFunc func() cfWorkflow.CfWorkflow,
-	recentLogsCmdGenerator, streamingLogsCmdGenerator, pushCmdGenerator cfCmdGenerator.CfCmdGenerator,
+	recentLogsCmdGenerator, streamingLogsCmdGenerator, pushCmdGenerator, appStatsCmdGenerator cfCmdGenerator.CfCmdGenerator,
```

Comment:
`appStatsCmdGenerator` is _after_ `pushCmdGenerator` here, but when `createMeasurements` is invoked above `appStatsCmdGenerator` is _before_ `pushCmdGenerator`.